spec/features/tag_importer_spec.rb

```diff
@@ -164,14 +164,19 @@ def and_it_has_been_marked_as_deleted
   end
 
   def then_i_can_see_it_is_ready_for_importing
+    old_path = current_path
```

Comment:
Can we just use the path helper for the page we mean? Makes it easier to follow.